### PR TITLE
[id] Update KubeCon dates for 2024

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/44158/files
/assign @habibrosyad

### CODE REVIEW

File: content/id/_index.html [hunk 1]
```diff
@@ -44,12 +44,12 @@ <h2>Tantangan yang Dihadapi untuk Melakukan Migrasi 150+ Microservices ke Kubern
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Watch Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" button id="desktopKCButton">Hadiri KubeCon North America pada 24-28 Oktober 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Hadiri KubeCon Europe pada 19-22 Maret 2024</a>
```
Comment (habibrosyad) [inline content/id/_index.html:47]: @Arhell Should it be KubeCon + CloudNativeCon? as in upstream EN

File: content/id/_index.html [hunk 2]
```diff
@@ -44,12 +44,12 @@ <h2>Tantangan yang Dihadapi untuk Melakukan Migrasi 150+ Microservices ke Kubern
...
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Hadiri KubeCon Europe pada 17-21 April 2023</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">Hadiri KubeCon North America pada 12-15 November 2024</a>
```
Comment (habibrosyad) [inline content/id/_index.html:52]: This one too

File: content/id/_index.html (hunk 1 above)
Comment (Arhell) [reply content/id/_index.html]: Done, thanks

File: content/id/_index.html (hunk 2 above)
Comment (Arhell) [reply content/id/_index.html]: Done, thanks

**Review by habibrosyad — APPROVED** [review]
Thanks!

/lgtm

**Pull request merged**